README.md

```diff
@@ -99,3 +99,23 @@ Instructions are the same as for dependencies (see above) with the following dif
 
 1. Directory must be created in `plugins/GEM_NAME`
 2. Plugins typically depend on `foreman` or `foreman-proxy`
+
+## HOWTO: compare packages
+
+Sometimes it is beneficial to compare the contents of this Git repository with the contents on deb.theforeman.org.
```

Comment:
If this is required after every package version bump then I dont think sometimes qualifies here?

Reply:
It's not required. You don't need this script to know something changed when you merge a PR.

It's just nice to sanity-check differences from time to time, e.g. for an release? ;)

Reply:
I'd say this should become procedure during stablization week and after branching. Other than that it can be useful when you make large changes.